examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -380,9 +380,9 @@ func (n *dcFnNamer) Name(t *types.Type) string {
 	pubName := n.public.Name(t)
 	n.tracker.AddType(t)
 	if t.Name.Package == n.myPackage {
-		return "DeepCopy_" + pubName
+		return "DeepCopy" + pubName
 	}
-	return fmt.Sprintf("%s.DeepCopy_%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
+	return fmt.Sprintf("%s.DeepCopy%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
```

Comment:
and yes lint is just suggestions but people run it, including kube, so we might as well follow thru on that thought.